addons/mail/static/src/core/web/activity_menu.js

```diff
@@ -55,11 +59,22 @@ export class ActivityMenu extends Component {
             // So, duplicates are faking the count and "Load more" doesn't show up
             force_search_count: 1,
         };
-        let domain = [["activity_user_id", "=", this.userId]];
+        let domain;
+        const isModelActivity = group.model === "mail.activity";
+        if (isModelActivity) {
+            domain = [
+                ["user_id", "=", this.userId],
+                ["id", "in", group.activity_ids],
```

Comment:
Isn't it redundant to have user_id = this.userId, as anyway the controller already filters them ? Shouldn't we use group.activity_ids only ?

Reply:
Up

Reply:
Yes indeed, that’s redundant. I have removed it.